conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake
+from conans.tools import load
+import re
+
+
+def get_version():
+    try:
+        content = load("CMakeLists.txt")
+        version = re.search(r"^\s*project\(ozo\s+VERSION\s+([^\s)]+)", content, re.M).group(1)
+        return version.strip()
+    except Exception:
+        return None
+
+
+class Ozo(ConanFile):
+    name = 'ozo'
+    version = get_version()
+    license = 'MIT'
```

Comment:
ozo has PostgreSQL License

Reply:
My bad, that's what you get from copying and pasting the recipe instead of writing it from the ground up.